### PR TITLE
Introduce "About thoughtbot" section to README.md

### PR DESCRIPTION
**Pull request by cpytel**
* Move credits into the contributing section
* Remove credits section
* Introduce "About thoughtbot" section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,20 +49,26 @@ Contributing
...
 factory_girl was originally written by Joe Ferris and is now maintained by Josh
 Clayton. Many improvements and bugfixes were contributed by the [open source
 community](https://github.com/thoughtbot/factory_girl/graphs/contributors).
 
-![thoughtbot](http://thoughtbot.com/assets/tm/logo.png)
+License
+-------
 
-factory_girl is maintained and funded by [thoughtbot, inc](http://thoughtbot.com/community)
+factory_girl is Copyright © 2008-2014 Joe Ferris and thoughtbot. It is free software, and may be redistributed under the terms specified in the [LICENSE](https://github.com/thoughtbot/factory_girl/blob/master/LICENSE) file.
```
Comment (croaky) [inline README.md]: 2015?

File: README.md [hunk 2]
```diff
@@ -49,20 +49,26 @@ Contributing
...
```
Comment (croaky) [inline README.md]: LICENSE can be relative path.

File: README.md [hunk 3]
```diff
@@ -49,20 +49,26 @@ Contributing
...
 
+About thoughtbot
+----------------
+
+![thoughtbot](https://thoughtbot.com/logo.png)
+
```
Comment (croaky) [inline README.md]: Newer version of logo at http://thoughtbot.github.io/presskit/

File: README.md [hunk 4]
```diff
@@ -49,20 +49,26 @@ Contributing
...
+factory_girl is maintained and funded by thoughtbot, inc.
 The names and logos for thoughtbot are trademarks of thoughtbot, inc.
 
-License
--------
+We are passionate about open source software.
```
Comment (croaky) [inline README.md]: "We love open source software!" ?

**Comment by croaky** [comment]
Merge at will.

File: README.md (hunk 3 above)
Comment (jferris) [inline README.md]: We're going to update `/logo.png` instead.

**Pull request closed without merging**